src/screens/SSILockScreen/index.tsx

```diff
@@ -14,23 +14,18 @@ import {IUser, RootState, ScreenRoutesEnum, StackParamList} from '../../types';
 interface IProps extends NativeStackScreenProps<StackParamList, ScreenRoutesEnum.LOCK> {
   users: Map<string, IUser>;
   login: (userId: string) => void;
+  onVerificationSuccess: () => Promise<void>;
```

Comment:
we already defined this in the stackparam props? so why is it declared here again? see the extends NativeStackScreenProps<StackParamList, ScreenRoutesEnum.LOCK>

Reply:
removed